crypto/des/asm/des-586.pl

```diff
@@ -15,7 +15,7 @@
 require "cbc.pl";
 require "desboth.pl";
 
-# base code is in microsft
+# base code is in microsoft
```

Comment:
Those are not in openssl-maintained code; those files are pulled in from krb5/krb5 for their external test suite.  (You could probably make a pull request for them there, though.)